hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -338,6 +340,14 @@ commit ctx st utxo = do
     (_, TxOut ShelleyAddressInEra{} _ _ _) ->
       Right ()
 
+  -- NOTE: Remove this limit once we have more experiments on mainnet.
+  rejectMoreThan100ADA :: NetworkId -> TxOut CtxUTxO -> Either (PostTxError Tx) ()
+  rejectMoreThan100ADA Mainnet output =
+    if selectLovelace (txOutValue output) > 100_000_000
```

Comment:
what if we create a constant to hold the limit amount `100_000_000` and reference it in the spec?

Reply:
I would say this is strongly encouraged (Should) to only hard-code the limit once and refer to it from here, the spec and any haddocks.